pmm-app/src/pmm-qan/panel/components/Details/Explain/Explain.styles.ts

```diff
@@ -2,18 +2,16 @@ import { css } from 'emotion';
 
 export const styles = {
   collapse: css`
-    background: #1f1d1d !important;
-    margin-bottom: 10 !important;
-    border: 1 !important;
-    border-color: white !important;
+    background: transparent !important;
```

Comment:
Do all those need to be `!important`?

Reply:
We need it to override antd styles it can be removed right after we remove antd